nsv13/code/modules/munitions/ammunition/torpedos/torpedo_types.dm

```diff
@@ -16,6 +16,13 @@
 	explode_when_hit = TRUE //Yeah, this can't ever end well for you.
 	var/claimable_gulag_points = 75
 
+/obj/item/ship_weapon/ammunition/torpedo/Initialize()
+	. = ..()
+	if(has_gravity())
+		move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	else
+		move_resist = MOVE_FORCE_DEFAULT
+
```

Comment:
Why is this in Initialize? Adjusting the move resist exclusively on creation depending on whether it had gravity when created seems somewhat counterproductive.
(Same for all other instances that have this placed in there)

Reply:
It was the only way I could figure out how to get it to drag when gravity turned off and when it turned back on again. Likewise I believe it was done like this to in case torps/missiles/nac were there roundstart since it didn't update right away when I was testing on a ship with a gravity generator that was off at roundstart.

Reply:
Maybe put it in can_be_pulled(), bumped() and move() instead? That way it's checked when any relative force is applied instead of only on initialize (which means you can currently still push it when gravity is re-enabled)
Also no need to add a comment for this in the torp file, it's pretty self-explainatory.